components/overview/index.js

```diff
@@ -183,7 +183,13 @@ const Overview = () => {
 				</span>
 				<button
 					className="border border-teal-500 text-teal-500 px-3 py-2 rounded-full"
-					onClick={() => toggle()}
+					onClick={() => 
+						isNil(accounts)
+							? router.push("/setup-wallet")
+							: selectedAccount
+							? toStaking()
```

Comment:
no flow leads to staking flow from overview
`isNil(accounts) ? router.push("/setup-wallet") : toggle()`
^ this should be the check for now

Reply:
Made the changes.